### ISSUE TITLE
pyro: giant error scroll on %fine

### ISSUE THREAD

**Issue by hosted-fornet** [uqbar-dao/dev-suite#86]
Related to #72: getting some more giant %pyro error scrolls that may be(?) harmless.

I can help you set this up so you can look into it; let me know when you've got time to take a look @tadad 

```hoon
%pyro-crash
~nec
/lib/pyro/vane/ames/hoon:<[1.828 60].[1.828 62]>
[%fine %request-events-forbidden]
/lib/pyro/vane/ames/hoon:<[1.828 23].[1.828 63]>
/lib/pyro/vane/ames/hoon:<[1.828 9].[1.840 53]>
/lib/pyro/vane/ames/hoon:<[1.827 9].[1.840 53]>
/lib/pyro/vane/ames/hoon:<[1.826 9].[1.840 53]>
/lib/pyro/vane/ames/hoon:<[1.825 9].[1.840 53]>
/lib/pyro/vane/ames/hoon:<[4.527 14].[4.527 53]>
/lib/pyro/vane/ames/hoon:<[4.525 5].[4.546 7]>
/lib/pyro/vane/ames/hoon:<[4.519 5].[4.546 7]>
/lib/pyro/vane/ames/hoon:<[4.516 5].[4.546 7]>
/lib/pyro/vane/ames/hoon:<[4.515 3].[4.548 20]>
/lib/pyro/vane/ames/hoon:<[4.513 3].[4.548 20]>
/lib/pyro/vane/ames/hoon:<[4.512 3].[4.548 20]>
/lib/pyro/vane/ames/hoon:<[4.510 3].[4.548 20]>
/lib/pyro/arvo/hoon:<[602 13].[602 28]>
/lib/pyro/arvo/hoon:<[602 6].[602 29]>
/lib/pyro/arvo/hoon:<[602 5].[602 37]>
/lib/pyro/arvo/hoon:<[601 5].[602 37]>
/lib/pyro/arvo/hoon:<[600 5].[602 37]>
/lib/pyro/arvo/hoon:<[624 5].[624 36]>
/lib/pyro/arvo/hoon:<[623 5].[624 36]>
/lib/pyro/arvo/hoon:<[618 5].[618 23]>
/lib/pyro/arvo/hoon:<[617 5].[618 23]>
/lib/pyro/arvo/hoon:<[616 5].[618 23]>
/lib/pyro/arvo/hoon:<[615 5].[618 23]>
/lib/pyro/arvo/hoon:<[1.173 25].[1.173 49]>
/lib/pyro/arvo/hoon:<[1.173 11].[1.174 21]>
/lib/pyro/arvo/hoon:<[1.171 11].[1.174 21]>
/lib/pyro/arvo/hoon:<[1.166 11].[1.174 21]>
call: failed
/lib/pyro/arvo/hoon:<[1.165 11].[1.174 21]>
/lib/pyro/arvo/hoon:<[1.164 11].[1.174 21]>
/lib/pyro/arvo/hoon:<[1.416 7].[1.417 42]>
bar-stack=[i=[i=//newt/0v1n.2m9vh t=~] t=~]
/lib/pyro/arvo/hoon:<[1.415 7].[1.417 42]>
/lib/pyro/arvo/hoon:<[1.414 7].[1.417 42]>
/lib/pyro/arvo/hoon:<[1.413 7].[1.417 42]>
/lib/pyro/arvo/hoon:<[1.407 7].[1.417 42]>
/lib/pyro/arvo/hoon:<[1.406 7].[1.417 42]>
/lib/pyro/arvo/hoon:<[1.303 9].[1.303 48]>
/lib/pyro/arvo/hoon:<[1.291 9].[1.303 48]>
/lib/pyro/arvo/hoon:<[1.289 9].[1.303 48]>
/lib/pyro/arvo/hoon:<[1.288 9].[1.303 48]>
/lib/pyro/arvo/hoon:<[1.287 9].[1.303 48]>
/lib/pyro/arvo/hoon:<[1.286 9].[1.303 48]>
/lib/pyro/arvo/hoon:<[1.281 7].[1.349 9]>
/lib/pyro/arvo/hoon:<[1.279 7].[1.349 9]>
/lib/pyro/arvo/hoon:<[1.277 7].[1.349 9]>
/lib/pyro/arvo/hoon:<[1.272 12].[1.272 22]>
/lib/pyro/arvo/hoon:<[1.272 7].[1.272 22]>
/lib/pyro/arvo/hoon:<[1.271 7].[1.272 22]>
/lib/pyro/arvo/hoon:<[1.270 7].[1.272 22]>
/lib/pyro/arvo/hoon:<[1.269 7].[1.272 22]>
/lib/pyro/arvo/hoon:<[1.267 7].[1.272 22]>
/lib/pyro/arvo/hoon:<[1.263 7].[1.272 22]>
/lib/pyro/arvo/hoon:<[1.262 7].[1.272 22]>
/lib/pyro/arvo/hoon:<[1.832 5].[1.832 45]>
/lib/pyro/arvo/hoon:<[1.831 3].[1.836 5]>
[%poke %hear]
/lib/pyro/arvo/hoon:<[1.830 3].[1.836 5]>
/lib/pyro/arvo/hoon:<[1.826 3].[1.836 5]>
/lib/pyro/arvo/hoon:<[1.818 3].[1.836 5]>
/app/pyro/hoon:<[228 16].[228 70]>
```

**Comment by tadad** [comment]
yeah I've gotten these in my normal dojo - I think part of it is just a rocky remote-scry rollout but worth looking into regardless

**Comment by hosted-fornet** [comment]
Okay, lets leave this alone for now, since it seems harmless and is just a lot of printing to the dojo.

**Comment by hosted-fornet** [comment]
Resolved by #90.
